### ISSUE TITLE
Token Deployment app uses different TokenReg contract than the Token Registry app

### ISSUE THREAD

**Issue by charlieknoll** [ethcore/parity#4027]
I created a new token using Token Deploy using the "PDF" TLA. I then used the Token Register app to register the contract created using Token Deploy. In the Token Registry app I made a mistake and didn't update the TLA to be "PDF". (I did not realize at first that the Token Deploy app registers a token to a TokenReg contract).

The Token Deploy tx is here:  https://etherscan.io/tx/0x81b129859b90877c0523c566b9ca545c37a0d00de099aad8e8bc3c7ecb015714 It used the TokenReg contract here: https://etherscan.io/address/0x87e093f4a39bc0f486f59e45af64d2d669d61b5b 

The TokenReg transaction is here: https://etherscan.io/tx/0xeb71304009d29eb40796658b4777319a7d51794504e7729dcb8f78d3eddf39e8 and it used the TokenReg contract here: https://etherscan.io/address/0x5f0281910af44bfb5fc7e86a404d0304b0e042f1 . 

I manually added both contracts and ABI to Parity and queried the "fromTLA" method. I found the registration using "PDF" as the TLA on the Token Deploy app's TokenReg and found the registration using "TLA" on the Token Registry app's TokenReg.

**Comment by jacogr** [comment]
It is by design.

You can manually register the token in the TokenReg dapp. Not optimal, but the option exists.

We actually do have a feature, which is not enabled atm in the deployment app, where the deployment will register with the main Registry. (Just taking it a bit slower here than normal since we didn't immediately want to flood the main Registry until we are sure we have all the issues ironed out.)

In short - we will enable this again shortly, it is a conscious decision made.
